change-beta/@azure-communication-react-9a04e371-cd36-43ab-b99a-bfc55df8702f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
Question for the gang, if this PR is running the stabilization workflow these changelogs should be `minor` right? since we are updating the stable API with this update?

Reply:
Changed it to minor